### PR TITLE
fix(ocm): parse bool instead of checking strings

### PR DESCRIPTION
**Pull request by shibumi**
This PR implements boolean parsing

### CODE REVIEW

File: pkg/ocm/ocm.go [hunk 1]
```diff
@@ -48,7 +49,12 @@ func New(ocmConfigFile string) (Client, error) {
 	// is only for local cadctl development
 	debugMode := os.Getenv("CAD_DEBUG")
 
-	if debugMode == "true" {
+	debugEnabled, err := strconv.ParseBool(debugMode)
+	if err != nil {
+		return client, fmt.Errorf("failed to parse CAD_DEBUG: %w", err)
```
Comment (georgettica) [inline pkg/ocm/ocm.go:54]: add the string provided by the user that caused the failute

**Review by georgettica — APPROVED** [review]
/lgtm
/approve

**Comment by shibumi** [comment]
@georgettica  shall we revert this PR and discuss this?

**Pull request merged**